examples/dt/adoption/rhv/control-plane.md

```diff
@@ -0,0 +1,38 @@
+# Configuring networking and deploy the OpenStack control plane
+This topology uses the copy of control-plane CR present in adoption repo.
+
+## Assumptions
+
+- A storage class called `local-storage` should already exist.
+
+## Initialize
+
+Switch to the "openstack" namespace
+
+```bash
+oc project openstack
+```
+
+Change to the adoption directory
+
+```bash
+cd data-plane-adoption/tests/config/base/
```

Comment:
I don't see this folder. Is it from a different source?

Reply:
yes, The source for controlplane CR for adoption is openstack-k8s-operators/data-plane-adoption repo. If the reference from other repo should not be in README.md then I can remove the steps referring different source repo.

Reply:
IMO, we should add that the repository is called to a said path as path of the `assumptions` section. Along with the update of change directory.

Reply:
Source repo added in Assumptions.